ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/spanning-tree.j2

```diff
@@ -10,22 +10,40 @@ STP mode: **{{ spanning_tree.mode | arista.avd.default("mstp") }}**
 
 STP Root Super: **{{ spanning_tree.root_super }}**
 {%     endif %}
+{%     if spanning_tree.root is arista.avd.defined %}
+
+STP Root: **{{ spanning_tree.root }}**
+{%     endif %}
 {%     if spanning_tree.mst_instances is arista.avd.defined %}
 
 ### MSTP Instance and Priority
```

Comment:
I would suggest changing this heading to `MSTP Instances` and then combine the two tables below, so it shows priority if set, and check for root primary/secondary if priority is not set. (Like done in config template)

Reply:
Maybe add the instance `vlans` to the table while we are at it. (move from below in another table)

Reply:
Done.